### ISSUE TITLE
Feature Suggestion: Display DAO params in dao home page, if connected wallet is DAO member

### ISSUE THREAD

**Issue by hard-nett** [DA0-DA0/dao-dao-ui#1727]
Can be easy to forget voting param setup while participating in many daos, so having a widget in the main app dao page only viewable by dao members might possibly help!

**Comment by noahsaso** [comment]
all voting params are visible on a DAO's home page right now. is there something missing?

**Comment by hard-nett** [comment]
hmmm, i do see the params when creating a new proposal, however from this page there isn't any details regarding the voting params
<img width="733" alt="image" src="https://github.com/DA0-DA0/dao-dao-ui/assets/123711748/dd3f8b4b-ce49-4207-a3e5-da6060c52aaf">

**Comment by noahsaso** [comment]
> hmmm, i do see the params when creating a new proposal, however from this page there isn't any details regarding the voting params <img alt="image" width="733" src="https://private-user-images.githubusercontent.com/123711748/311517720-dd3f8b4b-ce49-4207-a3e5-da6060c52aaf.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTAxMDYwNzcsIm5iZiI6MTcxMDEwNTc3NywicGF0aCI6Ii8xMjM3MTE3NDgvMzExNTE3NzIwLWRkM2Y4YjRiLWNlNDktNDIwNy1hM2U1LWRhNjA2MGM1MmFhZi5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwMzEwJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDMxMFQyMTIyNTdaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT0wOTVhOTUzMmJjMTgxMGU2YWJjYjYwMDc1ZTZmZTA5MmMzNGM3YmQwZTc3NWU2YjEyNTdlY2NhZjczNWNlMzY2JlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.H3Z_qPaiwPiMhaiQXLoE4xeXT7kXkwI0s19Tx3gYeDk">

ah yeah, we decided to move proposal-specific config to the proposal page because it was too much and cluttered the home page. is there specific information you want to see there?

**Comment by hard-nett** [comment]
yes maybe just having the current quorum & passing threshold neatly fit underneath the members & total supply details could be useful?

**Comment by noahsaso** [comment]
> yes maybe just having the current quorum & passing threshold neatly fit underneath the members & total supply details could be useful?

yeah, that is doable. i didn't do that because different proposal types (like single and multiple choice) may have different thresholds/quorums, and they work differently, so it may be a bit misleading to show on the homepage. but i can see why it would be useful there...
